pybaseball/datahelpers/postprocessing.py

```diff
@@ -1,30 +1,64 @@
 import re
-from typing import List, Union
+from datetime import datetime
+from typing import Dict, List, Union
 
 import numpy as np
 import pandas as pd
 
 null_regexes = [r'^\s*$', r'^null$']
+date_formats = [
+    '%Y-%m-%d',              # Standard statcast format
+    '%Y-%m-%dT%H:%M:%S.%fZ', # Just in case (https://github.com/jldbc/pybaseball/issues/104)
+]
+
+def try_parse_dataframe(
```

Comment:
Can we make this fit on one line? Maybe by setting something for `Unon[str, int, float', datetime']`?

Reply:
Define `ColType = Union[str, int, float', datetime']` somewhere, and annotate ColType?

Reply:
Do you find it less readable on multiple lines? I'll put it on one line if desired, but I find it more readable this way. You near instantly can see how many parameters it takes, of what type, and which have defaults. Whereas one one line they tend to somewhat run together in my opinion.

Thoughts?

Reply:
Also digging through the PEP to see what are the preferred way to handle this and found this discussion:

https://github.com/python/typing/issues/433

Where they suggested an alternative that would look like this if you'd prefer (though I personally find it less readable):

```python
def try_parse_dataframe(data: pd.DataFrame, null_replacement: Union[str, int, float, datetime] = np.nan,
                        known_percentages: List[str] = []) -> pd.DataFrame:
```